app/models/user.rb

```diff
@@ -34,4 +34,8 @@ def authenticated?(remember_token)
   def forget
     update_attribute(:remember_digest, nil)
   end
+
+  def feed
+    Task.where("user_id = ?", id)
```

Comment:
Usually you'll see this as `Task.where(user_id: id)`. We generally don't use this syntax unless we're doing something more complex. Although I'd say you don't need this method defined at all as you have the `tasks` association defined above. If you want it available as `feed` you could alias it:
```ruby
alias_attribute :feed, :tasks
```
https://api.rubyonrails.org/classes/ActiveModel/AttributeMethods/ClassMethods.html#method-i-alias_attribute

Reply:
This is perfect! Thank you so much! :D